src/LRS.js

```diff
@@ -875,8 +880,16 @@ TinCan client library
                 url: "activities/state",
                 method: "GET",
                 params: requestParams,
-                ignore404: true
+                ignore404: true,
+                headers: {}
             };
+
+            for (prop in optHeaders) {
+                if (optHeaders.hasOwnProperty(prop)) {
+                    requestCfg.headers[prop] = optHeaders[prop];
+                }
+            }
+
```

Comment:
Is there a reason to loop over the headers and set them into `requestCfg.headers` rather than just setting that property directly since we know from the change immediately above that it is already empty?